charts/rasa-x/templates/rasa-x-deployment.yaml

```diff
@@ -18,6 +18,9 @@ spec:
       labels:
         {{- include "rasa-x.selectorLabels" . | nindent 8 }}
         app.kubernetes.io/component: rasa-x
+      annotations:
```

Comment:
Sorry 🙈 But I think we also need to include the `rasa.checksum` here since Rasa X otherwise won't restart with the changes in the channels.

Reply:
👍